src/Designer/frontend/packages/shared/src/components/GiteaHeader/VersionControlButtons/components/BranchDropdown/BranchDropdown.module.css

```
@@ -4,6 +4,11 @@
   padding: 0 var(--ds-size-2);
 }
 
+.branchList {
+  max-height: 50vh; /*Fits screen at 150% scaling */
```

Stylelint (17.5.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)